tests/auth_tests/test_views.py

```diff
@@ -549,19 +550,19 @@ def test_security_check(self):
                         '///example.com',
```

Comment:
You could also put this data in separate tuples as other tests do as this indentation isn't the prettiest.